src/pf-utils/pf-utils.js

```diff
@@ -39,7 +43,7 @@ class PfUtil {
           return el;
         }
       } else if ( former === '.' ) {// If selector is a class
-        if ( new RegExp(latter).test(el.className) ) {
+        if ( this.hasClass(el, latter) ) {
```

Comment:
Why replace the RegExp?

Does this change break other components -- possibly relying on a regular expression?